canonical-web-and-design/ubuntu.com
# [security certs] Build landing page

## Done

- Build landing page at `/security/certifications`
- Add secondary nav (aware these pages don't exist yet but this is a feature branch)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/certifications
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit#) 
- This PR is made against the feature branch `security-certifications` - many of the links are pointing to pages that don't exist yet but will be built shortly 


## Issue / Card

Fixes #9747
